src/icon-tab-bar.scss

```diff
@@ -974,6 +974,131 @@ $fd-icon-tab-bar-semantic-values: (
     }
   }
 
+  &--navigation {
+    .#{$block}__header {
+      box-shadow: var(--sapShell_Shadow);
+      background: var(--sapShell_Navigation_Background);
+    }
+
+    .#{$block}__item {
+      color: var(--sapShell_Navigation_TextColor);
+    }
+
+    .#{$block}__tab {
+      .#{$block}__tag {
+        color: var(--sapShell_Navigation_TextColor);
```

Comment:
Do we really need nested selectors?
This provokes a prioritization war. It seems to me that this is not according to BEM. Now I can't just set a class and override the color. I need to give the appropriate priority to this selector otherwise it won't apply.
Sometimes nested selectors necessary, but I'm not sure if this is necessary in this case (maybe I'm wrong)

Reply:
yes, it's needed. It's due to the fact that `.fd-icon-tab-bar__tag` depends on the state of it's parent `.fd-icon-tab-bar__tab`. When `.fd-icon-tab-bar__tab` is selected, the color of the tag changes. Same is valid for the icon. 
```
 &__tab {
    @include fd-reset();

    @include fd-flex(column) {
      align-items: flex-start;
    }

    @include fd-selected() {
      @include fd-icon-tab-bar-selection-bar();
      ....
      .#{$block}__tag {
        color: var(--fdIcon_Tab_Bar_Selection_Color);
      }

      .#{$block}__icon {
        @include fd-icon-selector() {
          color: var(--sapContent_ContrastIconColor);
        }

        background-color: var(--sapSelectedColor);
      }
    }

```

For this reason, when we have the case with navigation we need to rely on the exact same specificity to change the color of the `.fd-icon-tab-bar__tag`